apps/docs/src/env.d.ts

```diff
@@ -1 +1 @@
-/// <reference types="astro/client" />
```

Comment:
i think both type imports are needed 👀

Reply:
I didn't know about the env.d.ts file, it was already a commited change when I've joined to `components-showcase` branch, done :)